torchbenchmark/models/pyhpc_equation_of_state/metadata.yaml

```
@@ -1,3 +1,6 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 16777216
```

```suggestion
    eval_batch_size: 1
```
In our sheet, its TFLOPS is always 0.